v20.1/create-user.md

```diff
@@ -25,6 +25,8 @@ The `CREATE USER` [statement](sql-statements.html) creates SQL users, which let
 
 The user must have the `INSERT` and `UPDATE` [privileges](authorization.html#assign-privileges) on the `system.users` table.
```

Comment:
This is no longer true - having CREATEROLE privilege is sufficient - this is true to all CREATE/ALTER/DROP role/user commands